requirements.txt

```diff
@@ -1,18 +1,19 @@
 setuptools>=59.4.0
 pyyaml>=6.0
 pycodestyle>=2.8.0
-netCDF4>=1.5.3
-matplotlib>=3.9.0
+netCDF4
```

Comment:
I'm not sure I like the idea of pinning versions of libraries in a generic requirements.txt. It could break the installs of Eva on different platforms.